arches_controlled_lists/datatypes/datatypes.py

```diff
@@ -28,8 +27,6 @@ class Reference:
 
 
 class ReferenceDataType(BaseDataType):
-    rest_framework_model_field = JSONField(null=True)
```

Comment:
I'm moving away from a pattern that expects this to exist on the python datatypes in favor of a fully isolated mapping so that API clients don't leak into the python datatypes.